### DISCUSSION TITLE
Can't install 17.2 on Ubuntu

### DISCUSSION THREAD

**Discussion by tresero** [gobuffalo/buffalo#2143] [Q&A] [answered]
Any clues?

Trying to install sqlite version on Ubuntu.

```
 go get -u -v -tags sqlite github.com/gobuffalo/cli/cmd/buffalo@0.17.2
go get: github.com/gobuffalo/cli/cmd/buffalo@0.17.2: invalid version: unknown revision 0.17.2
```

**Comment by paganotoni** [comment, marked as answer]
Hey @tresero 👋, 
I think you should add the `v` prefix on the version, e. g.:

```
go get -u -v -tags sqlite github.com/gobuffalo/cli/cmd/buffalo@v0.17.2
```

There is a PR to fix instructions on the site, I wonder if that caused confusion. LMK if that works.

**Reply by sio4** [reply]
For a record, @tresero mentioned that the fix worked: https://github.com/gobuffalo/buffalo/issues/2144#issuecomment-913281818
